articles/151_node_to_participant_mapping.md

```diff
@@ -0,0 +1,223 @@
+---
+layout: default
+title: Node to Participant mapping
+permalink: articles/node_to_participant_mapping.html
+abstract: This article analyzes the performance implications of enforcings a one-to-one mapping between ROS nodes and DDS participants, and propose alternative approaches.
+author: '[Ivan Paunovic](https://github.com/ivanpauno)'
+published: true
+categories: Middleware
+---
+
+{:toc}
+
+# {{ page.title }}
+
+<div class="abstract" markdown="1">
+{{ page.abstract }}
+</div>
+
+Original Author: {{ page.author }}
+
+## Introduction
+
+### What is a `Node`?
+
+In ROS, a `Node` is an entity used to group other entities.
```

Comment:
I don't know if `entities` is the best word for this. I used the same word used in DDS.